### ISSUE TITLE
Invalid configuration of the ExceptionHandling

### ISSUE THREAD

**Issue by SandGrainOne** [Altinn/altinn-file-scan#99]
### Description of the bug

The following message was registered in application insights in production:

> The exception handler configured on ExceptionHandlerOptions produced a 404 status response. This InvalidOperationException containing the original exception was thrown since this is often due to a misconfigured ExceptionHandlingPath. If the exception handler is expected to return 404 status responses then set AllowStatusCode404Response to true.

Looking into this we see that the ExceptionHandlier is pointed to a non existent endpoint. 

Program.cs L277
```
app.UseExceptionHandler("/filescan/api/v1/error");
```

We need to either ensure that the endpoint exists or find a solution that doesn't require an endpoint. Is there a default handler that can be used instead?

### Steps To Reproduce

Force an exception that isn't handled by the logic, but bobble up to the built in middleware.

**Comment by olebhansen** [comment]
Pending a decision on a prossible re-write/deprecation of file-scan

**Comment by TomasEng** [comment]
@olebhansen, does it exist an issue for this re-write/deprecation of file-scan?

**Comment by olebhansen** [comment]
@TomasEng : No. The re-write has stranded on getting clrity/decisions on requirements for filescanning from architects/security/platform. Will loop you in on the discussion.

**Comment by SandGrainOne** [comment]
@TomasEng There might be some remnants of some ideas as issues in the altinn-storage repo:
- https://github.com/Altinn/altinn-storage/issues/542
- https://github.com/Altinn/altinn-storage/issues/160

**Comment by TomasEng** [comment]
@olavsorl is taking over as the assignee for this issue. Let him know when there are any updates on the file scanning requirements, @olebhansen.
